code/_onclick/ai.dm

```diff
@@ -63,6 +66,10 @@
 		waypoint_mode = 0
 		return
 
+	if(controlled_mech) //Are we piloting a mech? Placed here so the modifiers are not overridden.
```

Comment:
RR, are you simply providing an example, or is that what you are suggesting as the solution?
@Aranclanos, would you find this an acceptable method?